docs/cluster_manifest.md

```diff
@@ -0,0 +1,215 @@
+  results in deploying or updating a completely separate cluster in the target
+  namespace. Optional (if present, should match the namespace where the
+  manifest is applied). 
+
+### Top-level parameters
+
+* **teamId**
+  name of the team the cluster belongs to. Changing it after the cluster
+  creation is not supported. Required field.
+
+* **dockerImage**
+  optional custom docker image that overrides the **docker_image** operator
+  parameter. Optional.
+
+* **enableMasterReplicaLoadBalancer**
+  boolean flag to override the operator defaults (set by the
+  `enable_master_load_balancer` parameter) to define whether to enable the load
+  balancer pointing to the postgres primary. Optional.
+
+* **enableReplicaLoadBalancer**
+  boolean flag to override the operator defaults (set by the
+  `enable_replica_load_balancer` parameter) to define whether to enable the
+  load balancer pointing to the postgres standby instances. Optional.
+
+* **allowedSourceRanges**
+  when one or more load balancers are enabled for the cluster, this parameter
+  defines the comma-separated range of networks (in CIDR-notation). The
+  corresponding load balancer is accessible only to the networks defined by
+  this parameter. Optional, when empty the load balancer service becomes
+  inaccessible from outside of the Kubernetes cluster.
+
+* **numberOfInstances**
+  total number of  instances for a given cluster. The operator parameters
+  `max_instances` and `min_instances` may also adjust this number.  Required
+  field.
+
+* **users**
+  a map of usernames to user flags for the users that should be created in the
+  cluster by the operator. User flags are a list, allowed elements are
+  `SUPERUSER` `REPLICATION` `INHERIT`, `LOGIN`, `NOLOGIN`, `CREATEROLE`,
+  `CREATEDB`, `REPLICATION`, `BYPASSURL`. A login user is created by default
+  unless NOLOGIN is specified, in which case the operator creates a role. One
+  can specify empty flags by providing a JSON empty array '*[]*'. Optional.
+
+* **databases**
+  a map of database names to database owners for the databases that should be
+  created by the operator. The owner users should already exist on the cluster
+  (i.e. mentioned in the `user` parameter). Optional.
+
+* **tolerations**
+  a list of tolerations that apply to the cluster pods. Each element of that
+  list is a dictionary with the following fields: `key`, `operator`, `value`,
+  `effect` and `tolerationSeconds`. Each field is optional. See [Kubernetes
+  examples](https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/)
+  for details on tolerations and possible values of those keys. When set, this
+  value overrides the `pod_toleration` setting from the operator. Optional.
+
+### Postgres parameters
+
+Those parameters are grouped under the `postgresql` top-level key.
+
+* **version**
+  the postgres major version of the cluster. Looks at the [Spilo
+  project](https://github.com/zalando/spilo) for the list of supported
```

Comment:
Maybe add reference to https://github.com/zalando/spilo/releases instead of just the repo itself?